hack/generate-docs.sh

```diff
@@ -27,12 +27,14 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 
 kube::golang::setup_env
 
-"${KUBE_ROOT}/hack/build-go.sh" \
-    cmd/gendocs \
-    cmd/genkubedocs \
-    cmd/genman \
-    cmd/genyaml \
-    federation/cmd/genfeddocs
+BINS=(
+	cmd/gendocs
+	cmd/genkubedocs
+	cmd/genman
+	cmd/genyaml
+	federation/cmd/genfeddocs
+)
+make -C "${KUBE_ROOT}" WHAT="${BINS[*]}"
```

Comment:
This demonstrates my lack of bash knowledge, but what is ${XXX[*]]. I've always used ${XXX[@]}.
What's the difference?